sensing/autoware_pointcloud_preprocessor/schema/dual_return_outlier_filter_node.schema.json

```diff
@@ -0,0 +1,141 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Dual Return Outlier Filter Node",
+  "type": "object",
+  "definitions": {
+    "dual_return_outlier_filter": {
+      "type": "object",
+      "properties": {
+        "x_max": {
+          "type": "number",
+          "description": "Maximum of x for `Fixed_xyz_ROI` mode",
+          "default": "18.0"
+        },
+        "x_min": {
+          "type": "number",
+          "description": "Minimum of x for `Fixed_xyz_ROI` mode",
+          "default": "-12.0"
+        },
+        "y_max": {
+          "type": "number",
+          "description": "Maximum of y for `Fixed_xyz_ROI` mode",
+          "default": "2.0"
+        },
+        "y_min": {
+          "type": "number",
+          "description": "Minimum of y for `Fixed_xyz_ROI` mode",
+          "default": "-2.0"
+        },
+        "z_max": {
+          "type": "number",
+          "description": "Maximum of z for `Fixed_xyz_ROI` mode",
+          "default": "10.0"
+        },
+        "z_min": {
+          "type": "number",
+          "description": "Minimum of z for `Fixed_xyz_ROI` mode",
+          "default": "0.0"
+        },
+        "min_azimuth_deg": {
+          "type": "number",
+          "description": "The left limit of azimuth for `Fixed_azimuth_ROI` mode",
+          "default": "135.0",
+          "minimum": 0
+        },
+        "max_azimuth_deg": {
+          "type": "number",
+          "description": "The right limit of azimuth for `Fixed_azimuth_ROI` mode",
+          "default": "225.0",
+          "maximum": 0
```

Comment:
You sure?
Add correct minimum and maximum to all angle related parameters...

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8475/commits/b122386103e1aae6985c89b30add9107e9fe077c